packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -563,6 +563,61 @@ const defaultConfigs = {
     uniswapAddress: "0xedf187890af846bd59f560827ebd2091c49b75df",
     twapLength: 7200,
     invertPrice: true
+  },
+  DIGGBTC: {
+    type: "expression",
+    // Note: lower-case variables are intermediate, upper-case are configured feeds.
+    expression: `
+      mean(DIGG_WBTC_SUSHI, DIGG_WBTC_UNI)
+    `,
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 1800,
+    priceFeedDecimals: 8,
+    customFeeds: {
+      DIGG_WBTC_SUSHI: { type: "uniswap", uniswapAddress: "0x9a13867048e01c663ce8ce2fe0cdae69ff9f35e3" },
+      DIGG_WBTC_UNI: { type: "uniswap", uniswapAddress: "0xe86204c4eddd2f70ee00ead6805f917671f56c52" }
+    }
+  },
+  DIGGETH: {
+    type: "expression",
+    // Note: lower-case variables are intermediate, upper-case are configured feeds.
+    expression: `
+      digg_wbtc = mean(DIGG_WBTC_SUSHI, DIGG_WBTC_UNI);
+      wbtc_eth = mean(WBTC_ETH_SUSHI, WBTC_ETH_UNI);
+      digg_wbtc * wbtc_eth
+    `,
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 1800,
+    priceFeedDecimals: 8,
+    customFeeds: {
+      DIGG_WBTC_SUSHI: { type: "uniswap", uniswapAddress: "0x9a13867048e01c663ce8ce2fe0cdae69ff9f35e3" },
+      DIGG_WBTC_UNI: { type: "uniswap", uniswapAddress: "0xe86204c4eddd2f70ee00ead6805f917671f56c52" },
+      WBTC_ETH_SUSHI: { type: "uniswap", uniswapAddress: "0xCEfF51756c56CeFFCA006cD410B03FFC46dd3a58" },
+      WBTC_ETH_UNI: { type: "uniswap", uniswapAddress: "0xBb2b8038a1640196FbE3e38816F3e67Cba72D940" }
+    }
+  },
+  DIGGUSD: {
+    type: "expression",
+    // Note: lower-case variables are intermediate, upper-case are configured feeds.
+    expression: `
+      eth_usd = 1 / USDETH;
+      digg_wbtc = mean(DIGG_WBTC_SUSHI, DIGG_WBTC_UNI);
+      wbtc_eth = mean(WBTC_ETH_SUSHI, WBTC_ETH_UNI);
+      digg_eth = digg_wbtc * wbtc_eth;
+      digg_eth * eth_usd
```

Comment:
Similarly, could we just use the other definitions?

We could also drop all of the customFeeds below in that case.

```suggestion
      eth_usd = 1 / USDETH;
      DIGGETH * eth_usd
```